lib/matplotlib/tests/test_backend_bases.py

```diff
@@ -114,7 +114,7 @@ def test_location_event_position(x, y):
         assert isinstance(event.y, int)
     if x is not None and y is not None:
         assert re.match(
-            "x={} +y={}".format(ax.format_xdata(x), ax.format_ydata(y)),
+            f"x={ax.format_xdata(x)} +y={ax.format_ydata(y)}",
```

Comment:
This is a less legible formatting, in my opinion. The symmetry of x and y is more easily seen without the f-string.